tests/async.test.tsx

```diff
@@ -1,8 +1,10 @@
-import React, { Fragment, StrictMode, Suspense, useRef, useEffect } from 'react'
+import React, { StrictMode, Suspense, useRef, useEffect } from 'react'
 import { fireEvent, render, waitFor } from '@testing-library/react'
 import { Provider as ProviderOrig, atom, useAtom, Atom } from '../src/index'
 
-const Provider = process.env.PROVIDER_LESS_MODE ? Fragment : ProviderOrig
+const Provider = process.env.PROVIDER_LESS_MODE
```

Comment:
Maybe extract this to a util function and use it everywhere?

Reply:
@dai-shi I'll do that and commit - Then you can revert if you disagree :-)

Reply:
I agree. Thanks for the suggestion.